MVVM/ViewModel/SafeMode_UIOverlay.cs

```diff
@@ -50,15 +60,15 @@ void UpdateDictionary()
 
 	void UpdateSafeModeIndicator()
 	{
-		if (_inputMode != (int)MqttClasses.ControlMode.Rover)
+		if (_inputMode != (int)MqttClasses.ControlMode.Rover && _inputModeSlave != (int)MqttClasses.ControlMode.Rover)
 		{
-			this.Visible = false; 
+			this.Visible = false;
 			return;
 		}
-		
+
```

Comment:
```suggestion

		this.GetNode<PanelContainer>("RoverMode").
			SetPosition(new Vector2((_inputMode == (int)MqttClasses.ControlMode.Rover) ? -2 : -191, 0));

		if(this.IsVisibleInTree()) return;

```

Moves safemode label to the correct place and prevents the safemode animation from replaying